### ISSUE TITLE
artifacts-ubuntu1804-test, artifacts-debian10-arm-test and artifacts-debian10-test failed due to systemd-timesyncd is inactive instead of activating it 

### ISSUE THREAD

**Issue by rayakurl** [scylladb/scylla#10455]
## Installation details

Cluster size: 1 nodes (n1-standard-2)

Scylla Nodes used in this run:
- artifacts-ubuntu1804-jenkins-db-node-ea949929-0-1 (34.138.6.71 | 10.142.0.105) (shards: -1)

OS / Image: `https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-1804-lts` (gce: us-east1)

Test: `artifacts-ubuntu1804-test`
Test id: `ea949929-ab2e-44b9-8fa7-cc4c453a13fd`
Test name: `scylla-master/artifacts/artifacts-ubuntu1804-test`
Test config file(s):

- [ubuntu1804.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/2c579884fe1fb585d972a93be302fd28d6035827/test-cases/artifacts/ubuntu1804.yaml)

## Issue description
During scylla setup, the ntp setup scripts goes over the possible time sync services, and if it finds any of them installed it activates it. In this test, the `systemd-timesyncd` service already existed in an inactive state. As part of the script, as seen here: https://github.com/scylladb/scylla/blame/master/dist/common/scripts/scylla_ntp_setup#L56 we check if the service is active, and if it's not it will be activated later (https://github.com/scylladb/scylla/blame/master/dist/common/scripts/scylla_ntp_setup#L84). However, the command that checks if the service is activated returns exit code 3 when the service is inactive, so the setup fails, when it should continue and attempt to activate it instead of failing the entire scylla setup.

```
Command: 'sudo /usr/lib/scylla/scylla_setup --nic ens5 --disks /dev/nvme0n1 --setup-nic-and-disks  --swap-directory / '
Exit code: 1
Stdout:
This is a supported kernel version.
Command 'systemctl  is-active systemd-timesyncd' returned non-zero exit status: 3
----------  stdout  ----------
inactive
------------------------------
----------  stderr  ----------
------------------------------
NTP setup failed.
```

```
0 | "program is running or service is OK" | unit is active
1 | "program is dead and /var/run pid file exists" | unit not failed (used by is-failed)
2 | "program is dead and /var/lock lock file exists" | unused
3 | "program is not running" | unit is not active
4 | "program or service status is unknown" | no such unit
```

- Restore Monitor Stack command: `$ hydra investigate show-monitor ea949929-ab2e-44b9-8fa7-cc4c453a13fd`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=ea949929-ab2e-44b9-8fa7-cc4c453a13fd)
- Show all stored logs command: `$ hydra investigate show-logs ea949929-ab2e-44b9-8fa7-cc4c453a13fd`

## Logs:
- **db-cluster-ea949929.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/ea949929-ab2e-44b9-8fa7-cc4c453a13fd/20220427_115257/db-cluster-ea949929.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ea949929-ab2e-44b9-8fa7-cc4c453a13fd/20220427_115257/db-cluster-ea949929.tar.gz)
- **sct-runner-ea949929.tar.gz** - [https://cloudius-jenkins-test.s3.amazonaws.com/ea949929-ab2e-44b9-8fa7-cc4c453a13fd/20220427_115257/sct-runner-ea949929.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ea949929-ab2e-44b9-8fa7-cc4c453a13fd/20220427_115257/sct-runner-ea949929.tar.gz)

[Jenkins job URL](https://jenkins.scylladb.com/job/scylla-master/job/artifacts/job/artifacts-ubuntu1804-test/526/)

**Comment by fruch** [comment]
Seem like the root cause if 48b6aec16a76f8cd60babf97cede99e7c0e50db1 @syuu1228 can you take a log, I think using the `out()` for `is-active` is the cause

**Comment by avikivity** [comment]
According to the commit changelog it was a 5.1 regression fixed in 5.1, so no backports needed.
